src/index.html

```diff
@@ -1,13 +1,22 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Antisnake</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Antisnake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Antisnake</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <main class="content">
+      <div class="container">
+        <div class="container__box container__box--color1">1</div>
+        <div class="container__box container__box--color2">2</div>
+        <div class="container__box container__box--color3">3</div>
+        <div class="container__box container__box--color4">4</div>
+        <div class="container__box container__box--color5">5</div>
+        <div class="container__box container__box--color6">6</div>
```

Comment:
```suggestion
        <div class="container__box">1</div>
        <div class="container__box">2</div>
        <div class="container__box">3</div>
        <div class="container__box">4</div>
        <div class="container__box">5</div>
        <div class="container__box">6</div>
```
You can use :nth-child for adding modifications.  Also, see below >¬↧